proto/osmosis/gamm/v1beta1/osmosis.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package osmosis.gamm.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "amino/amino.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "osmosis/poolmanager/v1beta1/swap_route.proto";
+
+option go_package = "github.com/Stride-Labs/stride/v14/x/stakeibc/types";
+
```

Comment:
* Delete swap_route.proto
* Delete swap_route.pb.go
* Move SwapAmountInRoute to here
```proto
message SwapAmountInRoute {
  uint64 pool_id = 1 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
  string token_out_denom = 2
      [ (gogoproto.moretags) = "yaml:\"token_out_denom\"" ];
}
```
* Update SwapAmountInRoute to use local
```proto
string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
  repeated SwapAmountInRoute routes = 2
      [ (gogoproto.nullable) = false ];
```